gammapy/background/energy_offset_array.py

```diff
@@ -24,7 +24,7 @@ class EnergyOffsetArray(object):
 
     """
 
-    def __init__(self, energy, offset, data=None):
+    def __init__(self, energy= None, offset= None, data=None):
```

Comment:
I add these None parameters in order to define empty EnergyOffsetArray() for the counts_2Dcube and livetime_2Dcube in the read method of EnergyOffsetBackgroundModel class as it is done in the CubeBackgroundModel class. However regarding how I define data in case of None value I will get an error if all the parameters are None so there are I think a better way to do it but it was just to start!